CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
+project(libaktualizr-demo-app)
 cmake_minimum_required(VERSION 3.8)
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/include)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/src/libaktualizr)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/third_party/jsoncpp/include)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/third_party/googletest/googletest/include)
+
+set(AKTUALIZR_VERSION "2020.8")
```

Comment:
Why is this necessary?

Reply:
Yeah, that's ugly. I was getting this
```
fatal: No names found, cannot describe anything.
CMake Error at aktualizr/CMakeLists.txt:106 (message):
Could not get current version from git
```
on the CI, and it seemed the easiest way to fix it.

Reply:
I think I was fixing something similar before, but can't remember the details.

Reply:
Interesting. Looks more like a libaktualizr issue. Thanks for checking!